### DISCUSSION TITLE
is there interest in distributing sbom file(s)

### DISCUSSION THREAD

**Discussion by tshepang** [vectordotdev/vector#19172] [General]
With the release of a usable `cargo-cyclonedx` crate (see https://www.reddit.com/r/rust/comments/17ud9it/cargocyclonedx_v040_now_ready_for_production), I would like to check if the project is interested in distributing SBOM files for Vector. I would do the work required to get it on CI, so that it becomes available on [GitHub Releases page](https://github.com/vectordotdev/vector/releases) for those who would make use of it (and I know at least one organisation that would).

**Comment by lfrancke** [comment]
Thank you! Yes, at least we'd be interested in having SBOMs available for Vector.
I think it'd make a useful addition.
